projects/vdk-plugins/vdk-plugin-control-cli/src/vdk/plugin/control_cli_plugin/vdk_plugin_control_cli.py

```diff
@@ -26,7 +26,9 @@
 from vdk.plugin.control_cli_plugin import control_service_configuration
 
 
-@hookimpl
+@hookimpl(
+    tryfirst=True
+)  # hook will be tried first so any hooks which decorate these commands will be able to function
```

Comment:
I don't understand the reason. If it's last , what would be the problem?

Reply:
The commands won't be registered under the click Group, so if anyone wants to alter those commands, they risk not having access to them. To give a more concrete example, imagine I'm trying to remove the prompt for a particular command parameter (hint hint). If this hook is executed before the one registering the command, then I won't be able to remove the prompt since I don't have access to it.

Reply:
Ok. Understood. My brain froze for a moment there. Maybe said that (but shorter) e.g "we add the commands first here so that they can be decorated later"

Reply:
noted, amended comment